ADALiOS/ADALiOS/ADKeychainTokenCacheStore.m

```diff
@@ -153,7 +153,7 @@ + (BOOL)handleKeychainCode:(OSStatus)errCode
 - (void)logItem:(ADTokenCacheStoreItem*)item
```

Comment:
I was referring to the token cache version here, just to prevent any issues that might pop up from reading older version cache.